packages/components/src/organisms/PriceRange/PriceRange.tsx

```diff
@@ -0,0 +1,192 @@
+import React, { useRef, useState, useImperativeHandle, forwardRef } from 'react'
+import type { AriaAttributes } from 'react'
+
+import { Price, Slider, InputField } from '../../index'
+import type { PriceProps, SliderProps } from '../../index'
+
+export type PriceRangeProps = Omit<SliderProps, 'absoluteValuesLabel'> & {
+  /**
+   * The current use case variant for prices.
+   */
+  variant?: PriceProps['variant']
+  /**
+   * Formatter function that transforms the raw price value and render the result.
+   */
+  formatter: PriceProps['formatter']
+  /**
+   * Returns the value of element's class content attribute.
+   */
+  className?: string
```

Comment:
Why we need this?

Reply:
Are we allowing the user to pass class to our components? 🤔 
I think I will remove it for now